models/issues/label.go

```diff
@@ -304,15 +304,12 @@ func GetLabelInRepoByName(ctx context.Context, repoID int64, labelName string) (
 		return nil, ErrRepoLabelNotExist{0, repoID}
 	}
 
-	l := &Label{
-		Name:   labelName,
-		RepoID: repoID,
-	}
-	has, err := db.GetByBean(ctx, l)
+	l, err := db.Get[Label](ctx, builder.Eq{"name": labelName, "repo_id": repoID})
```

Comment:
Something else we could also think about is refactoring `db.Get(…) (T, err)` to `db.Get(…) (T, has, err)`.
I think this would be even more user friendly as we then don't need to ignore the `ErrNotExist`, and thus less errors occur.

Reply:
Good idea.

Reply:
Done.